examples/update-demo/nautilus-rc.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ReplicationController
+id: update-demo-nautilus
+apiVersion: v1beta1
+desiredState:
+  replicas: 2
+  replicaSelector:
+    name: update-demo
+    version: nautilus
+  podTemplate:
+    desiredState:
+      manifest:
+        containers:
+        - name: update-demo
+          image: DOCKER_HUB_USER/update-demo:nautilus
+          ports:
+          - hostPort: 8080
```

Comment:
@roberthbailey mentioned in another example we shouldn't be using `hostPort` any longer. Does this apply here?